CMakeModules/Shaders.cmake

```diff
@@ -11,11 +11,9 @@
 
 # find_package(bgfx REQUIRED COMPONENTS shaderc)
 
-# get_target_property(BGFX_INCLUDE_PATH bgfx::bgfx INTERFACE_INCLUDE_DIRECTORIES)
-# set(BGFX_SHADER_INCLUDE_PATH ${BGFX_INCLUDE_PATH}/bgfx)
-
-add_executable(bgfx::shaderc ALIAS shaderc)
```

Comment:
This is needed if you are building bgfx from source

Reply:
I'll move this to a custom BuildBgfx.cmake file